openlibrary/plugins/openlibrary/api.py

```diff
@@ -397,7 +398,7 @@ def GET(self):
 
         # if bwb fails and isbn10, try again with isbn13
         if id_type == 'isbn_10' and \
-           metadata['betterworldbooks'].get('price') is None:
+                metadata['betterworldbooks'].get('price') is None:
             isbn_13 = isbn_10_to_isbn_13(id_)
```

Comment:
Let's please revert the formatting changes unrelated to the code here; it makes the git history really hard to read.